docs/windows.md

```diff
@@ -336,13 +262,18 @@ curl.exe -LO https://raw.githubusercontent.com/antrea-io/antrea/main/hack/window
 
 The script `Prepare-AntreaAgent.ps1` performs following tasks:
 
-* Prepare network adapter for kube-proxy.
+* Prepare network adapter for kube-proxy.(for Kubernetes versions prior to 1.26)
 
     kube-proxy needs a network adapter to configure Kubernetes Services IPs and
     uses the adapter for proxying connections to Service. Use following script
     to create the network adapter. The adapter will be deleted automatically by
     Windows after the Windows Node reboots.
 
+    Starting from Kubernetes 1.26, the kube-proxy userspace datapath is no longer
+    supported. For clusters running a version higher than 1.26, you should exclude
+    the preparation of the kube-proxy network adapter by executing the command
+    .\Prepare-AntreaAgent.ps1 -InstallKubeProxy $false.
```

Comment:
why aren't we making this the default behavior?

Reply:
I'm concerned if users refer to the documentation from a previous release, the script's default behavior may be different. If you don't see this as a problem, I can go ahead and change its default value. @antoninbas

Reply:
I understand what you mean. I think the problem may stem from the fact that the links to download the scripts are using the `main` branch:
```pwsh
curl.exe -LO https://raw.githubusercontent.com/antrea-io/antrea/main/hack/windows/Clean-AntreaNetwork.ps1
curl.exe -LO https://raw.githubusercontent.com/antrea-io/antrea/main/hack/windows/Prepare-ServiceInterface.ps1
curl.exe -LO https://raw.githubusercontent.com/antrea-io/antrea/main/hack/windows/Prepare-AntreaAgent.ps1
```
I think we can keep the `-InstallKubeProxy` as you currently have it. However, we should do something about these links so we don't have the same problem in the future.
We could do the same thing as for the `Helper.psm1` link earlier in the document:
```pwsh
curl.exe -LO "https://raw.githubusercontent.com/antrea-io/antrea/${TAG}/hack/windows/Helper.psm1"
```
We could also package these scripts as one zip file, and add it as a release asset. Not sure what the best solution is. The first one is probably the easiest, and we could make that change as part of this PR.

Reply:
Yes, we can update related links first.